signals/content/nlu_bec_high_conf.yml

```diff
@@ -0,0 +1,8 @@
+name: "NLU Intent: Business Email Compromise"
```

Comment:
do u want the confidence in the name like the others? missing on a few of em

Reply:
yeah good call.